actions/user_actions.jsx

```diff
@@ -299,18 +299,23 @@ export async function loadProfilesForDM() {
     }
 }
 
-export async function autocompleteUsersInTeam(username, success) {
-    const {data} = await UserActions.autocompleteUsers(username, getCurrentTeamId(getState()))(dispatch, getState);
-    if (success) {
-        success(data);
-    }
+export function autocompleteUsersInTeam(username, success) {
```

Comment:
Instead of still using the `success` callback, would it be difficult to change these to return the results of `doDispatch` and have the Providers treat this like a Promise? Most of the time that we use callbacks for async code, it's because that code was originally written before Promises were widely available.

Reply:
Hello @hmhealey , thanks for the feedback and pointing that out. I have made the changes.